### ISSUE TITLE
weex/vue/valid-image-component (eslint-plugin-weex bug)

### ISSUE THREAD

**Issue by GGICE** [weex-cli/vscode-weex-lang#5]
## Info

- Platform:  Win/macOS/Linux
- Vscode-weex-lang version: 0.0.13
- VS Code version: all 

## Problem
* `weex/vue/valid-image-component` not work
* `weex/vue/valid-image-component` show wrong warning
* `weex/vue/valid-image-component` only show once
*  other rule maybe same problem
 
![screen shot 2018-12-17 at 3 03 15 pm](https://user-images.githubusercontent.com/3450419/50071494-e9fc4d00-020c-11e9-9df6-1d21ced65d77.png)

## Reproducible Case

```
<template>
  <div class="wrapper">
    <image style="height:96px; width:96px;" :src="successIcon"/>
    <image style="height:100px; width:100px" :src="logo" class="logo"/>
    <image :src="logo" class="logo"/>
    <image style="height:96px; width:96px;" :src="successIcon"/>
  </div>
</template>
```

**Comment by GGICE** [comment]
Like this only show one error
```
<template>
  <div class="wrapper">
    <image style="height:96px;" :src="successIcon"></image>
    <image style="height:100px; width:100px" :src="logo" class="logo"></image>
    <image :src="logo" class="logo"></image>
    <image style="height:96px;" :src="successIcon"></image>
  </div>
</template>
```

**Comment by erha19** [comment]
It's been fixed with `eslint-plugin-weex@1.1.5`, please check @GGICE

**Comment by GGICE** [comment]
Version is eslint-plugin-weex@1.1.5
